### PR TITLE
Fjp/partial cell immersed boundaries

### PR DESCRIPTION
**Pull request by fadaie91**
Fixing the typo on defining the nodes for deltaz (c,c,c).

### CODE REVIEW

**Review by francispoulin — APPROVED** [review]
Thanks for doing this.

I'm approving it but maybe wait until Greg approves it before you merge into master?

**Comment by navidcy** [comment]
This PR is to be merged to [CliMA:fjp/partial_cell_immersed_boundaries](https://github.com/CliMA/Oceananigans.jl/tree/fjp/partial_cell_immersed_boundaries). So @francispoulin merge away I reckon!

**Comment by glwagner** [comment]
Right -- this is the correct way to update an existing PR from a fork. Well done @fadaie91 !

**Pull request merged**
